pkg/plugins/runtime/gateway/cluster_generator.go

```diff
@@ -163,13 +168,13 @@ func (c *ClusterGenerator) generateExternalCluster(
 func newClusterBuilder(
 	version envoy.APIVersion,
 	protocol core_mesh.Protocol,
-	dest route.Destination,
+	dest *route.Destination,
```

Comment:
Does the argument need to be mutated?

Reply:
Nope, but this is consistent with other apis.